### ISSUE TITLE
Internet Explorer Issue

### ISSUE THREAD

**Issue by Live4Development** [pojome/one-click-accessibility#31]
I found an issue when using Internet Explorer 11. There is excessive white spacing within the containers. I believe this is because of the icons not having a height assigned to them. For example:

`#pojo-a11y-toolbar .pojo-a11y-toolbar-overlay ul.pojo-a11y-toolbar-items li.pojo-a11y-toolbar-item a svg{height: 10px;}`

![Accessability-Tool-Plugin-Internet-Explorer-Issue---July-2019](https://user-images.githubusercontent.com/5461399/61884721-baac6d80-aeca-11e9-8971-168d67c5d76f.jpg)

**Comment by besweeet** [comment]
Observing the same issue.

**Comment by Rodrigoleon** [comment]
Same issue.

**Comment by XhmikosR** [comment]
Yeah, IE needs width and height attributes to show the SVGs properly.
